spec/Appendix B -- Grammar Summary.md

```diff
@@ -45,7 +45,7 @@ Token ::
   - FloatValue
   - StringValue
 
-Punctuator :: one of ! $ & ( ) ... : = @ [ ] { | } . [lookahead != .]
+Punctuator :: one of ! $ & ( ) ... : = @ [ ] { | } . [lookahead != {`.`, Digit}]
```

Comment:
Good catch for decimals, but does the lookahead apply to the entire list or only the `.`? If the entire list, this currently forbids `foo: Int = 9` I think?